docs/_static/stravalib.css

```diff
@@ -2,6 +2,11 @@ html, body {
     font-size: 1.02rem;
 }
 
+/* We are using comments for code output so make them more visible */
+html[data-theme="dark"] .highlight .go {
```

Comment:
This doesn't work - in dark mode comments are really light.i was trying to brighten them a but but i may have to experiment a bit more.